docs/virtual_machines/lifecycle.md

```diff
@@ -45,6 +45,17 @@ corresponding `VirtualMachineInstance` object using `kubectl`.
 > deleted from the cluster. You will not be able to start this
 > VirtualMachineInstance object again.
 
+## Starting and stopping a virtual machine
+
+Virtual machines, in contrast to VirtualMachineInstances, have a running state. Thus on VM you can define if it
+should be running, or not. VirtualMachineInstaces are, if they are defined in the cluster, always running and consuming resources.
```

Comment:
@fabiand Typo in `VirtualMachineInsta(n)ces`: [...]VirtualMachineInstances are, [...]